src/kyc.ts

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+} from './'
+import { MarqetaError } from './'
```

Comment:
I think this is just a `type`, and if so, it can go in the `import type` above...